README.md

```diff
@@ -1,83 +1,80 @@
+implementation of the mode. Emacs and SuperCollider each have their own package
+managers, so it is required to install each half separately.
 
-For the HTML help system, you will need emacs-w3m support.
+### Installing scel quark
 
+The `scel` Quark is required for emacs to communicate with sclang. Evaluate this in the SuperCollider GUI:
 
-Installation (default)
-----------------------
-
-By default emacs-lisp files are installed in
-
-`$prefix/share/emacs/site-lisp`
-
-SuperCollider files are put in
+``` supercollider
+Quarks.install("https://github.com/supercollider/scel");
+```
 
-`$prefix/share/SuperCollider/Extensions/scide_scel`
+### Installing the emacs mode
 
+Using straight.el
+``` emacs-lisp
+(straight-use-package
+ '(sclang :type git 
+          :host github 
+          :repo "supercollider/scel" 
+          :files ("el/*.el")))
+```
 
-The only thing you need to do is loading the sclang interface in your `~/.emacs`:
+Or download the repo directly to your user config directory
 
+``` shell
+git clone https://github.com/supercollider/scel.git ~/.emacs.d/scel
 ```
+``` emacs-lisp
+(add-to-list 'load-path "~/.emacs.d/scel/el/")
 (require 'sclang)
 ```
 
-For the HTML help system to fully function also add
-```
-(require 'w3m)
+### On MacOS
+
+If `sclang` executable is not on your path, you may need to add it to your exec-path.
+
+``` emacs-lisp
+(setq exec-path (append exec-path '("/Applications/SuperCollider.app/Contents/MacOS/")))
 ```
+### On Linux
 
+If you are building SuperCollider from source on Linux, this library (both .el
+and .sc files) will be installed by default. To disable it pass the flag
+`-DSC_EL=OFF` as a `cmake` option. See the supercollider readme for more info.
 
-Installation (detailed)
------------------------
 
-Put all `*.el` files in emacs' load-path. e.g. if you put them in
-`~/emacs/`, add the following lines to `~/.emacs` (or whatever your init
-file is called):
+## Installation requirements
 
-```
-(add-to-list 'load-path "~/emacs")
-(require 'sclang)
-```
+For the HTML help system, you will need emacs-w3m support, but you can still use without that.
 
-for the HTML help system to fully function also add
-```
+```emacs-lisp
 (require 'w3m)
 ```
 
-now put all `*.sc` files in sclang's library path, e.g. if you put them
-in a non-standard location, such as `~/SuperCollider/Emacs`, add the
-following to `~/.config/SuperCollider/sclang_conf.yaml` (Linux) or `~/Library/Application Support/SuperCollider/sclang_conf.yaml` (macOS):
+## Configuration
 
-```
-includePaths:
-    [~/SuperCollider/Emacs]
-```
+To fine-tune the installation from within emacs' graphical customization interface, type:
 
-(note normally this is not needed as they are put into sclang's library
-path during installation with scons).
+`M-x sclang-customize`
 
 
-Usage
------
+## Usage
 
-In order to automatically start sclang when invoking emacs, use the following command line:
+`M-x sclang-start` or open a `.scd` file and press `C-c C-o`
```

Comment:
Again, newbie comment: we need to be in `sclang-mode` first, so that needs to be activated for new buffers.